internal/config/config.go

```diff
@@ -97,7 +106,9 @@ func LoadConfig(filePath string, fileName string) (*Config, error) {
 	v.SetDefault("kubelet.networkRouteFile", "/proc/net/route")
 	v.SetDefault("nodeName", "node")
 	v.SetDefault("nodeIP", "node")
-	v.SetDefault("httpServerPort", 0)
```

Comment:
This is set from an env var in the chart and due to 
```
	// We need to assure that defaults have been set in order to bind env variables.
	// https://github.com/spf13/viper/issues/584
```
we still need to have a default for that and change their  name if we want to be able to customize it from envs.

(however it is true that currently we do not strictly need different ports since only control plane has hostNetwork=true)

Reply:
we can sync offline about this

Reply:
I completely forgot. I am bringing this line back.